source/contracts/Controller.sol

```diff
@@ -127,26 +124,11 @@ contract Controller is IController {
         return true;
     }
 
-    function switchModeSoOnlyEmergencyStopsAndEscapeHatchesCanBeUsed() public devModeOwnerOnly returns (bool) {
+    function switchToCentralizedMode() public devModeOwnerOnly returns (bool) {
         whitelist[owner] = false;
```

Comment:
Should this use `Controller.removeFromWhitelist` instead of editing `whitelist` directly?

Reply:
Yea that makes sense. Will change